src/config/networks.ts

```diff
@@ -37,5 +37,12 @@ export const defaultNetworks: NetworksConfig = {
     proxyFactoryAddress: '0xfC7577774887aAE7bAcdf0Fc8ce041DA0b3200f7',
     multiSendAddress: '0x8D29bE29923b68abfDD21e541b9374737B49cdAD',
     fallbackHandlerAddress: '0x40A930851BD2e590Bd5A5C981b436de25742E980'
+  },
+  // xdai
+  100: {
+    masterCopyAddress: '0x6851D6fDFAfD08c0295C392436245E5bc78B0185',
```

Comment:
Why do we not use the official mastercopy on xdai?

Reply:
Wait, those are not the official addresses? I got them from https://github.com/gnosis/safe-contracts/blob/development/networks.json

Reply:
Ahhh you use the v1.2.0 address ... then I would update this everywhere, not just for xdai. You should check by tag

Reply:
It's fine. The rest of the networks are updated here: https://github.com/gnosis/contract-proxy-kit/pull/126